schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-sidenav-container>
+  <mat-sidenav fixedInViewport="true" [mode]="isHandset ? 'over' : 'side'" #drawer [opened]="!(_isHandset | async)!.matches">
+    <mat-toolbar color="primary">Menu</mat-toolbar>
+    <mat-nav-list>
+      <a mat-list-item href="#">Link 1</a>
+      <a mat-list-item href="#">Link 2</a>
+      <a mat-list-item href="#">Link 3</a>
+    </mat-nav-list>
+  </mat-sidenav>
+  <mat-sidenav-content>
+    <mat-toolbar color="primary">
+      <button type="button" mat-icon-button (click)="drawer.toggle()" *ngIf="(_isHandset | async)!.matches">
```

Comment:
We need to make sure the a11y here is sensible. We may need to add a focus-trap to the sidenav (which in turn might need `role="dialog"` added to the sidenav element along with everything that goes along with that role)